### DISCUSSION TITLE
Applying prisma migration command to an existing database without altering the current state of the database

### DISCUSSION THREAD

**Discussion by byamasu-patrick** [prisma/prisma#18615] [Q&A]
Hello, I am trying to apply the migration on the existing database that I have hosted on digital ocean so that I should be able to ship my application but I encountering with this error. I have searched the documentation but I can't find the right answer that give me exactly what I want. The connection string is working perfectly fine, and my database on using Postgresql but if I try to run this command without applying the migration because I don't need to alter my actual database state But it failing. All I want to is have add new changes of what I have in my prisma schema without modifying the state of the production database. I hope anyone can help. Thanks 

` npx prisma migrate dev --name initial-migration --create-only`

```
Error: P3014

Prisma Migrate could not create the shadow database. Please make sure the database user has permission to create databases. Read more about the shadow database (and workarounds) at https://pris.ly/d/migrate-shadow

Original error:
db error: ERROR: permission denied to create database
   0: migration_core::state::DevDiagnostic
             at migration-engine\core\src\state.rs:269

```

**Comment by janpio** [comment]
> Prisma Migrate could not create the shadow database. Please make sure the database user has permission to create databases. Read more about the shadow database (and workarounds) at https://pris.ly/d/migrate-shadow

The error message is pretty explicit what the problem is, and includes a link to a docs page that explains how to work around it.

**Reply by ludralph** [reply]
Hi @byamasu-patrick, In addition to @janpio answer, you might want to specifically check this section of the [docs](https://www.prisma.io/docs/concepts/components/prisma-migrate/shadow-database#cloud-hosted-shadow-databases-must-be-created-manually) for creating a shadow database in a cloud-hosted environment.

**Reply by nurul3101** [reply]
@byamasu-patrick If this answers your question, it would be great if you could **mark this Discussion as answered** to indicate that it has been resolved. 

Otherwise please let us know how else we can help you further or close the Discussion if it was resolved in some other way 🙏
